com.github.scrivanolabs.scrivano.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.scrivanolabs.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-config
```

Comment:
```suggestion
```

Why would you need that?

Reply:
My reasoning was that if the user is using both a Flatpak version and a non Flatpak version of Scrivano, they would be able to use access the same configuration files. However, this is probably not wise and I'll commit your suggestion.

Reply:
This is not the standard practice at all. You also grant access to just other app preferences.

It's not like we can verify the source code at any point.